Branch-SDK/Branch-SDK/BNCServerInterface.m

```diff
@@ -228,4 +233,38 @@ - (BNCServerResponse *)processServerResponse:(NSURLResponse *)response data:(NSD
     return serverResponse;
 }
 
+- (void)updateDeviceInfoToMutableDictionary:(NSMutableDictionary *)dict {
+    BNCPreferenceHelper *preferenceHelper = [BNCPreferenceHelper preferenceHelper];
+    BOOL isRealHardwareId;
+    NSString *hardwareId = [BNCSystemObserver getUniqueHardwareId:&isRealHardwareId andIsDebug:preferenceHelper.isDebug];
+    if (hardwareId) {
+        dict[BRANCH_REQUEST_KEY_HARDWARE_ID] = hardwareId;
```

Comment:
can you grab `ad_tracking_enabled` as well: https://github.com/BranchMetrics/iOS-Deferred-Deep-Linking-SDK/blob/master/Branch-SDK/Branch-SDK/Requests/BranchInstallRequest.m#L48